docker/nvidia/setup_nvidia_docker_compose.py

```diff
@@ -56,6 +56,7 @@ def get(self, endpoint):
 else:
     # from NVIDIA public hub repository https://hub.docker.com/r/nvidia/cuda/
     cuda_pkg_versions = {
+        '9.0': '9-0=9.0.176-1',
```

Comment:
so if someone passes `--cuda-version=9.0` then this is the version that will get downloaded from the NVIDIA docker hub?

Reply:
ok - so our images already have these versions. This just tells us which one to pick?

Reply:
ok makes sense

Reply:
No they don't.  We augment our base image and commit into a new image.